command.go

```diff
@@ -38,11 +43,32 @@ func substitute_variables(params string, variables map[string]string) string {
 }
 
 func (c *Command) MustRun() bool {
-	return c.Enabled && (c.lastRun.IsZero() || time.Now().Sub(c.lastRun) >= c.Interval)
+
+	buffer := time.Second * time.Duration(60*60*2) // setting a 2 hour buffer
```

Comment:
Just use the natural formulation of time durations, such as
```go
buffer := 2 * time.Hour
```
then there's no need for a comment explaining "2 hour"